app/javascript/components/pages/About.js

```diff
@@ -1,7 +1,98 @@
 import React from 'react'
+import { Card, Button, Row } from 'react-bootstrap'
+import { NavLink } from 'react-router-dom'
+import sean1 from "../assets/sean1.jpg"
+import michael1 from "../assets/michael1.jpg"
+import don1 from "../assets/don1.jpeg"
+import valerie1 from "../assets/valerie1.jpg"
 
 export default function About() {
   return (
-    <div>About</div>
+    <>
+      <br />
```

Comment:
Has no semantic meaning, therefore you a better practice would be to use css positioning instead.

Reply:
Will do Eddie. I'll go over when we're doing a style overhaul and remove all the <br/>